platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md

```diff
@@ -316,6 +316,22 @@ You bring:
   - **TestRail** [TestRail VSP accessibility test plan template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc) **OR**
   - **Markdown** [Github accessibility test cases template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/accessibility-test-cases-template.md)
 - List of known issues (with links to tickets when applicable)
+- On the request template, certify that the following accessibility checks are complete:
```

Comment:
I'm not sure what this request template represents. I think it's related to the previous bullet point about including a link to the TestRail or Markdown a11y template. If that's the case, how about moving this above the bullet about list of known issues.

Reply:
The request template is the Staging Review request issue template a team submits when they are ready for a staging review.

I attempted not to change the structure of the documentation much from what VSP had, apart from adding the checklist to clarify for teams who currently are overwhelmed by trying to learn TestRail and telling me they don't understand the accessibility test cases template either.